data/en/2_0001.json

```diff
@@ -5,13 +5,28 @@
         "blocks": [{
                 "type": "Introduction",
                 "id": "introduction",
-                "information_to_provide": [
-                    "Employee numbers"
-                ],
-                "basis_for_completion": [
-                    "Please complete this statutory questionnaire for {{respondent.trad_as_or_ru_name}}, for the period {{exercise.start_date|format_date}}.",
-                    "Data should relate to all sites in England, Scotland and Wales, unless otherwise stated in the ‘trading as’ name, above."
-                ]
+                "body": [{
+                    "id": "basis-for-completion",
+                    "type": "Basic",
+                    "content": [{
+                        "title": "Basis for completion",
+                        "list": [
+                            "Please complete this statutory questionnaire for {{respondent.trad_as_or_ru_name}}, for the period {{exercise.start_date|format_date}}.",
+                            "Data should relate to all sites in England, Scotland and Wales, unless otherwise stated in the ‘trading as’ name, above."
+                        ]
+                    }]
+                }, {
+                    "id": "information-to-provide",
+                    "type": "Basic",
+                    "content": [{
+                        "title": "You will be asked to provide information for the business, including:",
+                        "list": [
+                            "Employee numbers"
+                        ]
+                    }, {
+                        "description": "<p class=\"venus\">If actual figures are not available, please provide informed estimates.</p>"
```

Comment:
You'll lose having that piece of text as a member of the the typography hierarchy; so changes to the hierarchy wont impact it.

The <p> adds some extra whitespace but if you mean doing <p><strong>If actual...</strong></p> then there'd be no visible difference.